app/helpers/data.py

```diff
@@ -202,9 +203,24 @@ def add_session_to_event(request, event_id, state=None, use_current_user=True):
         :param event_id: Session belongs to Event by event id
         """
         form = request.form
-        slide_file = DataManager.get_files_from_request(request, 'slides')
-        video_file = DataManager.get_files_from_request(request, 'video')
-        audio_file = DataManager.get_files_from_request(request, 'audio')
+        slide_temp_url = form.get('slides_url')
+        video_temp_url = form.get('video_url')
+        audio_temp_url = form.get('audio_url')
+        slide_file = ''
+        video_file = ''
+        audio_file = ''
+
+        if slide_temp_url:
+            slide_file = UploadedFile('{}/static/{}'.format(path.realpath('.'), slide_temp_url[len('/serve_static/'):]),
```

Comment:
@shubham-padia instead of this, could you use the `get_path_of_temp_url` method from `app/helpers/assets/images.py` ?